lib/proxy/request.rb

```diff
@@ -7,6 +7,7 @@ module Proxy::HttpRequest
   class ForemanRequestFactory
     def initialize(base_uri)
       @base_uri = base_uri
+      @base_uri += '/' if @base_uri[-1..-1] != '/'
```

Comment:
I look at 1.8.7 docs, String#end_with? is available: http://ruby-doc.org/core-1.8.7/String.html#method-i-end_with-3F